packages/publisher/github/src/PublisherGithub.ts

```diff
@@ -101,25 +99,35 @@ export default class PublisherGithub extends PublisherBase<PublisherGitHubConfig
             uploaded += 1;
             updateUploadStatus();
           };
-          const artifactName = path.basename(artifactPath);
-          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          if (release!.assets.find((asset: OctokitReleaseAsset) => asset.name === artifactName)) {
-            return done();
+          if (!release) {
+            throw new NoReleaseError(404);
+          }
+          const artifactName = GitHub.sanitizeName(artifactPath);
+          const asset = release.assets.find((item: OctokitReleaseAsset) => item.name === artifactName);
+          if (asset !== undefined) {
+            if (config.force === true) {
```

Comment:
Nit(style): is the check explicitly for `true` strictly necessary here? What would be the downside for a truthy check?

Reply:
We could change this up in the PR IMO. It would be a bit cleaner. We already use explicit boolean checks elsewhere in the Publisher, but inconsistently (we use `config.draft !== false` and `config.prerelease === true`, which have opposite effects for passing in a truthy non-boolean value).

Reply:
Feel free to commit this suggestion :smile: 

```suggestion
            if (config.force) {
```